src/OmniSharp.MSBuild/ProjectFile/PackageReference.cs

```diff
@@ -5,21 +5,20 @@ namespace OmniSharp.MSBuild.ProjectFile
 {
     public class PackageReference : IEquatable<PackageReference>
     {
-        public PackageIdentity Identity { get; }
+        public PackageDependency Identity { get; }
```

Comment:
maybe you'd want to go from `public PackageIdentity Identity { get; }` to `public PackageDependency Dependency { get; }` instead?